### ISSUE TITLE
Problems with the pre-train model  'imagenet-resnet-50-dag.mat'

### ISSUE THREAD

**Issue by tjulyz** [vlfeat/matconvnet#504]
When I use the 'cnn_imagenet_googlenet.m' to load the 'imagenet-googlenet-dag.mat' to classify a picture which I download from the internet, I can get the right result. But when I use the same function to load the 'imagenet-resnet-50/101/152-dag.mat', the result is absolute irrelated. So, is there anything wrong you can help me?  Thanks!

**Comment by lenck** [comment]
Hi, because the resnet contains batchNormalization layers, you need to switch the network to test mode:
```
net.mode = 'test' ;
```
By default it is in the `normal` mode which does not disable the batchNormalization.

**Comment by tjulyz** [comment]
Thank you very much！ It works now! When I classify some pictures, the result is not so well as the googlenet. I thingk I should try more. Thanks again!

**Comment by lenck** [comment]
Hi, that is weird... Have you tried to run evaluation on the whole ILSVRC12? Andrea was generating the table on the MatConvNet website using [this script](https://github.com/vlfeat/matconvnet/blob/master/utils/evaluate_ref_models.m) so if you are not able to obtain similar results there must be some issue...

**Comment by tjulyz** [comment]
I think I should try more. If there still are something wrong, I will contact you again. Thanks for your reply.

**Comment by lenck** [comment]
Ok, if you would find some problems, feel free to reopen this issue :)
